purchase_order_report_with_note/report/purchase_report.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <report id="purchase.report_purchase_quotation"
+        string="Request for Quotation"
+        model="purchase.order"
+        name="purchase_order_report_with_note.report_purchasequotation_qweb"
+        report_type="qweb-pdf"
+        menu="True"
+    />
+    <report id="purchase.report_purchase_order"
+        string="Purchase Order"
+        model="purchase.order"
+        name="purchase_order_report_with_note.report_purchaseorder_qweb"
+        report_type="qweb-pdf"
+        menu="True"
+    />
```

Comment:
Could be a way to rewrite this where this becomes a proper inherit from the original view instead of overwriting it?

Runbot doesn't like this